Version 3.8.0 release notes (#57)
MapsIndoors/mapsindoors.github.io: collections/_cms/changelog.md

published: true
---

## January 2020 - v3.8.0

### What’s New:

> * **CAD file upload:** Ability to upload a CAD file (.dwg, .dxf) per floor in the CMS. You can find a guide in the documentation https://mapsindoors.github.io/cms/#buildings.

Reviewer: ```suggestion
* **CAD file upload:** Ability to create new buildings and floors and upload CAD files (.dwg, .dxf) for the floors in the CMS. You can find [a guide in the documentation](https://mapsindoors.github.io/cms/#buildings).
```

> * **Icon Manager:** Added the option to “skip” uploading icons larger than 128x64px.

Reviewer: ```suggestion
* **Icon Manager:** Added the option to "skip" uploading icons larger than 128x64px.
```

>   * When deleting a used icon from the Icon Manager, it is also removed from the map.

Reviewer: ```suggestion
  * When deleting an icon in the Icon Manager that is in use on the map, it is also removed from the map.
```

  
### Bug Fixes:

> * **Icon Manager:** shows a more descriptive error message when an image fails.

Reviewer: ```suggestion
* **Icon Manager:** Shows a more descriptive error message when an image fails.
```

* **Locations:** Google Street View network now available inside The White House demo solution.

> * **Route Access:** Fixed issue where eternal doors showed up as elevator doors.

Reviewer: ```suggestion
* **Route Access:** Fixed issue where external doors showed up as elevator doors.
```

I really like the idea of _eternal doors_, but I think you meant _external_ 😉

Author: 😄
